client/reducers/index.js

```diff
@@ -0,0 +1,10 @@
+import { combineReducers } from 'redux';
+import loginReducer from './loginReducer';
+import usersReducer from './usersReducer';
+import crudReducer from './crudReducer';
+
+export default combineReducers({
+    crudReducer,
+    loginReducer,
+    usersReducer,
+});
```

Comment:
this is the root reducer, it grabs all of the other reducers and combines them. 

when a redux action is dispatched, it passes that action to all of the reducers, they are then handled in the `switch` statements to distribute the new state accordingly. as you can see in all of the reducer files, state is NEVER mutated directly, we always return a NEW object.